web_ui/src/core/labels/label.interface.ts

```diff
@@ -9,6 +9,7 @@ interface LabelCommon {
     readonly hotkey?: string;
     readonly behaviour: LABEL_BEHAVIOUR;
     readonly isEmpty: boolean;
+    readonly isBackground: boolean;
```

Comment:
We should no longer need this field with the addition of `BACKGROUND` to `LABEL_BEHAVIOUR`.
For any code that needs to check if a label is a background label we should be able to do `isBackgroundLabel(label)` which returns it based on the label behaviour.

Reply:
I think we should remove it there as well. So essentially we should be able to determine all of the label's behaviour from its behaviour field.
Tbh I'm not entirely sure why we also have the `isEmpty` field while that can be derived from `behaviour` as well. That might be something we can look into in the future?

Reply:
removed